app/models/comment.rb

```diff
@@ -1,2 +1,9 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+
+  belongs_to :post, class_name: 'Post', foreign_key: :post_id
+
+  def update_comment_count
+    post.increment!(:comments_counter)
```

Comment:
Optional: It would be nice to use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_comments_counter method every time a new post has been saved to database.

Example:
`after_save :update_comments_counter`